test

```diff
@@ -59,3 +59,4 @@ test coroutines-co.js
 test coroutines-bluebird.js
 test await.compiled.js
 test tasks.js
+test kgo.js
```

Comment:
I'd like to group the callback-based solutions and the promise-based solutions. At the very least, we need to include a link to __kgo.js__ in the README. :)

Please use `git commit --amend` to avoid introducing a second commit.